packages/hooks/src/useInvoiceDetails.ts

```
@@ -1,19 +1,204 @@
+import { useMemo } from 'react';
+import { Hex } from 'viem';
 import { useBalance } from 'wagmi';
 
-import { useInstantDetails } from './useInstantDetails';
-import { useIpfsDetails } from './useIpfsDetails';
-import { useTokenData } from './useTokenMetadata';
+import {
+  fetchInstantInvoice,
+  fetchTokenBalance,
+  fetchTokenMetadata,
+} from './helpers';
```

ESLint

[error] Missing file extension for "./helpers"

(import/extensions)